manuscript/draft.Rmd

```diff
@@ -283,83 +283,88 @@ biodiversity in the form of species richness. The results of this effort are
-richness next year than it did last year there is little confidence that this
-will be the case. This results in simple site-level time-series and baseline
-models like the long-term average and the naive model being some of the best
-performing overall models for future biodiversity.
-
-The most commonly used method to forecast the future state of biodiversity,
-fitting a species distribution model separately to each species and then summing
-the probability of presence across those models, provides the worst overall
-forecasts of all methods evaluated. Most forecasting based on this approach is
-never evaluated and when it is it is typically only assessed based on cross-site
-rather than within site approaches. As a result the forecasts may appear to
-perform well even when change through time is not accurately predicted. This
-issue has also recently been recently highlighted as an issue for using SDMs to
-predict the future locations of individual species [plos one 2012 paper]. This
-result is particularly sobering because this method forms the foundation for
-major claims regarding the predicted loss of biodiversity to climate change
-[2004 science/nature papers]. The poor performance of SDM-based forecasts
-highlights the crucial importance of comparing multiple modeling approaches when
-conducting ecological forecasts, and in particular the value of comparing
-results to simple baselines to avoid over-interpreting the accuracy of
-ecological forecasts. Disciplines that have more mature forecasting cultures
-often do this by reporting "forecast skill", which is the improvement in the
-forecast relative to a simple baseline. We recommend adopting this approach in
-future ecological forecasting research. It also highlights the need for using
-hindcasting and time-series data for assessing the effectiveness of forecasting
-methods, since this allows comparisons to baselines and assessment of the
-predicted dynamics at each site.
+future, relative to other sites, it generally does. However, the forecasts
+perform relatively poorly at predicting variation in richness over time for any
+given site. If a site is predicted to have a higher richness next year than it
+did last year there is little confidence that this will be the case. This
+results in simple site-level time-series, which are best predicted by simple
+baselines like the long-term average and the "naive" model.
+
+While stacked species distribution models are the most commonly used approach
+for forecasting the future state of biodiversity [[cite?]], this approach
+provided the worst overall forecasts of all methods evaluated. In fact, the most
+common version of the stacking approach (based on rounding species' occurrence
+probabilities up or down before adding them up) would have led to even worse
+results than what we presented [Pellissier et al. 2013's "probabilistic
+approach" paper, Calabese et al. 2014]. In general, SDM-based forecasts are
```

Comment:
@sdtaylor I guess this depends on how we're thinking about rarely. My general impression is that there are on the order of only about 10 dynamic assessments of SDM forecasts, compared to thousands of papers making them. That feels "rare" to me, but if that doesn't seem like the right word to you I'm certainly open to suggestions.